reactioncommerce/reaction
# Fix #3371 Test Shopify credentials before saving.

### Fix for #3371 

### To test
1. Reset database to load new transaltions `reaction reset -n`
1. Run `reaction`
2. Open the Connector from Action menu.
3. Enter invalid API credentials and hit the `Save` button. You should get a message that setting was *not* saved.
4. Enter valid API credentials and hit the `Save` button. You should get a message that setting was saved.

### Review
Should the message on invalid credetials be something like "Invalid Credentials" or just say "Setting not saved"(when behind the scene they actually have been saved)?
